### ISSUE TITLE
Uncaught MemoryError for large export (IDA 32bit)

### ISSUE THREAD

**Issue by dwendt** [joxeankoret/diaphora#30]
Hey, seems like there's operations on ~100mb bins that can cause the DB to get too big to fit entirely in memory. It'd be cool if these exceptions were caught instead of losing the few hours of exporting, but I've been trying to think of a way to check if the next insert will hit the memcap and I just really dunno...(ensure db is updated on disk and check (filesize * some overhead modifier)?

Specific error:
```
Error: 
Traceback (most recent call last):
  File "O:/reversing/diaphora-master/diaphora.py", line 3074, in _diff_or_export
    bd.export()
  File "O:/reversing/diaphora-master/diaphora.py", line 1221, in export
    self.do_export()
  File "O:/reversing/diaphora-master/diaphora.py", line 1211, in do_export
    self.save_function(props)
  File "O:/reversing/diaphora-master/diaphora.py", line 1082, in save_function
    cur.execute(sql, new_props)
MemoryError
```

**Comment by joxeankoret** [comment]
Uhm... first time I see such a problem exporting. Can you share an IDB?

**Comment by joxeankoret** [comment]
Here is my PGP key, just in case: http://joxeankoret.com/contact.html

**Comment by joxeankoret** [comment]
I added an attempt to fix this bug with 44f5b9bcadea8b33dfe517ad4de765af266cdedf. I think it fixed the issues but, naturally, without an IDB that used to fail I cannot probe.

**Comment by joxeankoret** [comment]
Closing as I have no feedback. Fill free to re-open it again if you encounter it again.
